lib/game_loop.rb

```diff
@@ -0,0 +1,53 @@
+require_relative './guess'
+
+class GameLoop
+
+	attr_reader :past_guesses, :has_won
+	
+	GUESS_LIMIT = 10
+	WIN_CONDITION = 4
+
+	def initialize(game_state, dialog, input)
+		@valid = true
+		@game_state = game_state
+		@dialog = dialog
+		@input = input
+	end
+
+	def play
+		while (@game_state.remaining_guesses > 0)
```

Comment:
`@game_state.remaining_guesse.positive?`

Reply:
For some reason when I try .positive?, it works when I run the program but fails with my tests. Maybe the version of rspec that I have is out of date?